python/ray/experimental/ui.py

```diff
@@ -263,6 +264,72 @@ def handle_submit(sender):
     task_search.on_submit(handle_submit)
 
 
+# Helper function that guarantees unique and writeable temp files.
+# Prevents clashes in task trace files when multiple notebooks are running.
+def _get_temp_file_path(**kwargs):
+    temp_file = tempfile.NamedTemporaryFile(delete=False,
+                                            dir=os.getcwd(),
+                                            **kwargs)
+    temp_file_path = temp_file.name
+    temp_file.close()
+    return os.path.relpath(temp_file_path)
+
+
+# Helper function that ensures that catapult is cloned to the correct location
+# and that the HTML files required for task trace embedding are in the same
+# directory as the web UI.
+def _setup_trace_dependencies():
+    catapult_home = "/tmp/ray/catapult"
+    catapult_commit = "33a9271eb3cf5caf925293ec6a4b47c94f1ac968"
+    try:
+        # Check if we're inside a git repo
+        cmd = ["git",
+               "-C",
+               catapult_home,
+               "rev-parse",
+               "--is-inside-work-tree"]
+        subprocess.check_call(cmd)
+
+    except subprocess.CalledProcessError:
+        # Error on non-zero exit code (e.g. - ".git not found")
+        pass
```

Comment:
Then I'm not sure I understand the purpose of this check. If the `subprocess.check_call` raises an exception (i.e., we're not in a git repo), then `_setup_trace_dependencies` won't succeed, right? But we're suppressing the error.

Or am I misunderstanding it? What's the point of checking if we're in the git repo if we're going to ignore the answer?